scripts/check_no_audit_scratch_scripts.sh

```
@@ -0,0 +1,91 @@
+fi
+
+# Fail OPEN on a parse error: this hook is narrow defense-in-depth (the Phase 7
+# sweep is the backstop), so a jq hiccup must never block a legitimate write.
+FILE=$(jq -r '.tool_input.file_path // ""' 2>/dev/null) || exit 0
+[[ -z "$FILE" ]] && exit 0
+
+norm() {
+    local p="${1//\\//}"
+    if [[ "$p" =~ ^([A-Za-z]):/(.*)$ ]]; then
+        local drive
+        drive=$(printf '%s' "${BASH_REMATCH[1]}" | tr 'A-Z' 'a-z')
```

Shellcheck (0.11.0)

[info] Use '[:upper:]' to support accents and foreign alphabets.

(SC2019)